src/Microsoft.OpenApi/CallerArgumentExpressionAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Runtime.CompilerServices;
+
+[ExcludeFromCodeCoverage]
+[DebuggerNonUserCode]
+[AttributeUsage(AttributeTargets.Parameter)]
+sealed class CallerArgumentExpressionAttribute :
+    Attribute
+{
+    public CallerArgumentExpressionAttribute(string parameterName) =>
+        ParameterName = parameterName;
+
+    public string ParameterName { get; }
```

Comment:
isn't the value here "value" in case of netstandard 2.0?

Reply:
i have just kept the same convention as the actual implementation https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.callerargumentexpressionattribute.-ctor?view=net-7.0#system-runtime-compilerservices-callerargumentexpressionattribute-ctor(system-string)

Reply:
I guess my confusion is how does this custom attribute get the actual parameter name since it doesn't have the support of the framework and can't take advantage of compiler capabilities? (due to the version being targeted)

Reply:
> can't take advantage of compiler capabilities

thats not correct. compiler capabilities are determined by the c# version being targeted. some times u need the marker attributes for some features

Reply:
oh this is only compile time sugar! I thought this relied on some IL feature. Thanks for clarifying.